### PR TITLE
Fix r2r (the sh script)

### PR DESCRIPTION
**Pull request by kazarmy**
Faster to type `./r2r`  than `node ...` .

### CODE REVIEW

File: r2r [hunk 1]
```diff
@@ -31,6 +31,8 @@ control_c() {
 
 trap control_c 2
 
+R2R="node node_modules/node-r2r/bin/r2r.js"
```
Comment (radare) [inline r2r]: nope. thats not the bin you have to run.

use `$(npm bin)/r2r`

Comment (radare) [reply r2r]: which his basically node_modules/.bin/r2r

Comment (kazarmy) [reply r2r]: > node_modules/.bin/r2r

^ please (ac03e54868df6e63b52b4086cababcbab8100b7e) ... calling `npm bin` takes an extra second or so

**Comment by radare** [comment]
wtf buy a new machine. takes 0.2s here, anyway its the only portable way to do that, but 99% of the cases will be in node_modules/.bin/r2r, not in node-r2r/bin

**Comment by kazarmy** [comment]
upgraded npm and time to execute `npm bin` went up to around 3 seconds (!) ... anyway 19cb6c38ce0092ccc0a750157f737727c249a1db uses `npm bin` as requested

**Pull request merged**